src/botPage/bot/Interpreter.js

```diff
@@ -15,6 +16,7 @@ export default class Interpreter {
     init() {
         this.$scope = createScope();
         this.bot = new Interface(this.$scope);
+        this.blockly = new _Blockly();
```

Comment:
You don't need to import blockly, instead, you can use `this.stop()` to stop the interpreter, which in turn stops the UI. Look at [this](https://github.com/binary-com/binary-bot/pull/452/files#diff-a01567c526c81e4c69f7321eda89e0faL92)